src/mapping.ts

```diff
@@ -38,19 +41,19 @@ registerNodeFactory(ParserRuleContext, () => new GenericParseTreeNode());
 //Augment the ParseTree class with a toAST method
 declare module 'antlr4ts/tree' {
     export interface ParseTree {
-        toAST(parent?: Node): Node;
+        toAST(parent?: Node): Node | undefined;
```

Comment:
In realtà qui, poiché sono metodi, abbiamo la garanzia che `this` non sia né null né undefined, quindi possiamo lasciare la signature precedente...